packages/core/src/SDKCore/SDKCore.ts

```diff
@@ -111,28 +110,23 @@ export class SDKCore {
   }
 
   get isLoggedIn() {
-    if (!this.at_exp) {
-      return false;
-    }
-
```

Comment:
@JakeLo123 `isLoggedIn()` feels like it should always return a bool no?

Reply:
Yes, it will return a bool. I removed this `if(!this.at_exp) { return false }` guard because I switch `at_exp` from type `number | null` to `number` where it's just `-1` if it's not present.